### PR TITLE
Updates to docs files

### PR DESCRIPTION
**Pull request by Charles1992Sail**
Following on from recent file changes, I am recommending some grammatical changes focused on improving readability for a more natural flow. 

My writing style guide is The Publication Manual of the American Psychological Association, Seventh Edition (2020).

I will revisit the docs later in order to catch those spots I missed.  My main goals were:

- I focused on changing the second-person “you” to the third-person “user”.  The APA Manual lacks provisions on the use of the second-person pronoun (American Psychological Association, 2020, §4.16- §4.21, pp. 118-123).

- I focused on consistent punctuation (American Psychological Association, 2020, Introduction through §6.10, pp. 153-161).

American Psychological Association. (2020). Publication manual of the American Psychological Association: The official guide to the APA style (7th ed.). http://doi.org/10.1037/000016-000

American Psychological Association. (n.d.). Publication manual of the American Psychological Association, seventh edition (2020). https://apastyle.apa.org/products/publication-manual-7th-edition

### CODE REVIEW

File: docs/sbom-tool-arguments.md [hunk 1]
```diff
@@ -1,107 +1,107 @@
...
+
+The Sbom tool generates a SBOM for any build artifact.
+
+
+GlobalOption    Description
+Help (-?, -h)   Prints this help message
+
+Actions
+
+  Validate -options - Validate a build artifact using the manifest. Additional optio which verifies the signing certificate of the manifest.
```
Comment (adriandiglio) [inline docs/sbom-tool-arguments.md:17]: Typo. Please change "optio" to "option"

**Review by Charles1992Sail — COMMENTED** [review]
Spelling changes.

File: docs/building-from-source.md [hunk 2]
```diff
@@ -1,3 +1,71 @@
+<<<<<<< Updated upstream
```
Comment (JamieMagee) [inline docs/building-from-source.md:1]: This looks like a git conflict market

File: docs/building-from-source.md [hunk 3]
```diff
@@ -64,3 +132,4 @@ docker build . -t ms_sbom_tool
 ```
 
 Use docker bind mounts when using the tool to scan the desired target directories.
+>>>>>>> Stashed changes
```
Comment (JamieMagee) [inline docs/building-from-source.md:135]: This looks like a git conflict marker

**Comment by Charles1992Sail** [comment]
Closing pull request and will resubmit a new fresh one.

**Pull request closed without merging**